build.xml

```diff
@@ -220,7 +220,7 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle">
 
     <property name="netty.version" value="3.10.6.Final"/>
 
-    <property name="junit.version" value="4.8.1"/>
+    <property name="junit.version" value="4.12"/>
```

Comment:
This version change comes from master/3.5. Do we really intend to upgrade to JUnit version 4.12 in this PR?

Reply:
Yes, I need this to use assertNotEquals().

Reply:
Ok